.rubocop.yml

```diff
@@ -9,6 +9,9 @@ AllCops:
     - db/**/*
     - bin/**/*
 
+RSpec/InstanceVariable:
+  Enabled: false
+
```

Comment:
I feel like this rule has kept me from experiencing unintentional side effects between examples.  The tests look the same if you use memoized values, but you get the expected value at the start of each example instead of something that might have gotten altered through some side effect.

Reply:
That's why this is a bad rule. if you use a before( :context) block, to only run fixture setup once per suite, you cannot use let(), and must use an instance variable. this comes in handy in the case where you're setting up an expensive fixture (such as creating something in fedora!), that you don't want to constantly tear down between specs.  this is why the violation says "prefer, when possible". in the csv_importer spec, it's not desirable to create a new Collection for every spec, as it takes around 2s each time.  One should indeed be careful though.